src/th/reapertrans/src/eu/kanade/tachiyomi/extension/th/reapertrans/ReaperTrans.kt

```diff
@@ -0,0 +1,12 @@
+package eu.kanade.tachiyomi.extension.th.reapertrans
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ReaperTrans : MangaThemesia(
+    "ReaperTrans",
+    "https://www.reapertrans.com",
+    "th",
+    dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale("th")),
```

Comment:
Site is using English months.

Reply:
Fixed

![Screenshot_20240817_182719_Mihon](https://github.com/user-attachments/assets/4a894684-1289-41cf-845f-76d483a4aaae)